src/data/witchess_solutions.txt

```diff
@@ -0,0 +1,151 @@
+# Puzzle Number	Solution
```

Comment:
this file should have a version number at the top (i.e. the first line should be "1").